Automattic/wp-calypso
# WooCommerce: Action List - Product Create

This PR adds the Product Create operation for the ~~API plan~~ Action List code.
It's based on the API plan code in #14236 

To Test:

1. `npm test` and ensure all tests pass.
2. Visit http://calypso.localhost:3000/store/product/<site url>
3. Fill out some product info and click "Save"
4. Notice should pop up that product has been saved.
5. Confirm on `wp-admin` that product was created.
